internal/profile/_static/docker-compose-stack.yml

```diff
@@ -78,6 +78,7 @@ services:
     hostname: docker-fleet-server
     environment:
     - "FLEET_SERVER_ENABLE=1"
+    - "FLEET_SERVER_POLICY_ID=fleet-server-managed-ep"
```

Comment:
this should be removed if 7.x versions are not going to have the new policies

Reply:
We can try to set `FLEET_SERVER_POLICY_ID` to an empty string for 7.x. Do you think it will be sufficient? I'm trying not to fork the `docker-compose-stack.yml`.

Reply:
this envvar is not necessary at all if using the `is_default_fleet_server` flag in 8 config